Update managing-holds.md (#8554)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/managing-holds.md

12. Review your settings, and then click **Create this hold**.

> [!NOTE]

> > When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. Once the character count across all queries on a single location exceed 10,000 characters, the entire location is placed on hold. 

Reviewer: ```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. Once the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```

> > When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. Once the character count across all queries on a single location exceed 10,000 characters, the entire location is placed on hold. 

Reviewer 2: ```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```

> > When you create a query-based hold, all content from selected locations is initially placed on hold. Once the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 

Reviewer 2: ```suggestion
> When you create a query-based hold, all content from selected locations is initially placed on hold. After the timer job in either Exchange or SharePoint runs, any content that doesn't match the specified query is cleared from the hold. After the character count across all queries on a single location exceeds 10,000 characters, the entire location is placed on hold. 
```
